lib/doh.c

```diff
@@ -937,46 +938,42 @@ UNITTEST void de_cleanup(struct dohentry *d)
 CURLcode Curl_doh_is_resolved(struct connectdata *conn,
                               struct Curl_dns_entry **dnsp)
 {
+  CURLcode result = CURLE_OK;
   struct Curl_easy *data = conn->data;
   *dnsp = NULL; /* defaults to no response */
 
-  if(!data->req.doh.probe[0].easy && !data->req.doh.probe[1].easy) {
+  if(!data->req.doh.probe[DOH_PROBE_SLOT_IPADDR_V4].easy
+     && !data->req.doh.probe[DOH_PROBE_SLOT_IPADDR_V6].easy) {
```

Comment:
We usually end the first line with the operator on the far right end when we split them

Reply:
OK, de gustibus ...